artscraper/find_artworks.py

```diff
@@ -209,26 +209,46 @@ def get_artist_works(self):
         # Find the parent element corresponding to the text heading
         parent_element = element.find_element('xpath', '../..')
 
-        # Find right arrow button
-        right_arrow_element = parent_element.find_element('xpath', \
-            './/*[contains(@data-gaaction,"rightArrow")]')
-
-        # Check if right arrow button can still be clicked
-        while right_arrow_element.get_attribute('tabindex') is not None:
+        # Find number of artists
+        # Find elements with tag name 'h3'
+        items_elements = parent_element.find_elements('tag name', 'h3')
+        for element in items_elements:
+            if 'items' in element.text:
+                match = re.search(r'\d+', element.text)
+                if match:
+                    total_num_artworks = int(match.group())
+                    break
+
+        # Initialize number of artworks
+        num_artworks = 0
+
+        while num_artworks < total_num_artworks:
```

Comment:
What if the num_artworks never reaches total_num_artworks? It will then simply loop endlessly right? Would be good in that case to have a maximum number of tries (with the same results).

Reply:
I now make it try thrice before breaking out of the loop.